lib/progressTracker.js

```diff
@@ -26,54 +26,60 @@ function track (instance, opts) {
   const iOpts = instance.opts
   if (opts.renderProgressBar) {
     let msg = `${iOpts.connections} connections`
+
     if (iOpts.pipelining > 1) {
       msg += ` with ${iOpts.pipelining} pipelining factor`
     }
-    if (!iOpts.amount) {
-      const progressBar = new ProgressBar(opts.progressBarString, {
-        width: 20,
-        incomplete: ' ',
-        total: iOpts.duration,
-        clear: true,
-        stream: opts.outputStream
-      })
 
+    if (!iOpts.amount) {
       logToStream(`Running ${iOpts.duration}s test @ ${iOpts.url}\n${msg}\n`)
 
-      progressBar.tick(0)
-
-      instance.on('done', () => progressBar.tick(iOpts.duration - 1))
-      instance.on('tick', () => {
-        progressBar.tick()
-      })
-
-      process.once('SIGINT', () => {
-        progressBar.tick(iOpts.duration - 1)
-      })
+      // if switch needed needed to avoid
+      // https://github.com/mcollina/autocannon/issues/60
+      if (opts.outputStream.isTTY) {
```

Comment:
We could turn the progress bar output into its own function, and in that case, we could return early if the output stream isn't a tty, like below

```js
if (!opts.outputStream.isTTY) return
```

Not a worry if you don't want to, only a minor thing I realised